maison-de-l-orient-et-de-la-mediterranee.csl

```diff
@@ -0,0 +1,200 @@
+    </names>
+  </macro>
+  <macro name="pages">
+          <text variable="page"/>
+     </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter=", "/>
+      <substitute>
+        <names variable="editor" />
+        <names variable="translator" />
+      </substitute>
+    </names>
+  </macro>
+    <macro name="access">
+ <text variable="URL" prefix=", disponible sur : " suffix=" "/>
+    <group>
+      <text term="accessed" prefix="["/>
+      <date variable="accessed"  suffix="]">
+        <date-part name="month" suffix=" " prefix=" " />
+        <date-part name="year"/>
+      </date>
+    </group>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="webpage" match="any">
+        <text variable="title"/>
+      </if>
+      <else-if variable="container-title" match="none">
+        <text variable="title" font-style="italic"/>
+      </else-if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter="" suffix=", ">
+      <text prefix=" (" macro="edition" suffix=") "/>
+      <text variable="collection-title"/>
+      <text variable="collection-number" prefix=" "/>
+    </group>
+    <text variable="publisher-place"/>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short" suffix=", " text-case="lowercase"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="loc-cite">
+   <group delimiter="">
+      <label variable="locator" form="short"/>
+      <text variable="locator"/>
+   </group>
+  </macro>
+  <citation and="text" et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=",">
+    <sort>
+      <key variable="issued"/>
+    </sort>
+    <layout prefix=" (" suffix=")" delimiter=" ; ">
+      <text macro="author-short" suffix=" "/>
+      <text macro="year-date"/>
+      <group>
+        <text macro="loc-cite" prefix=", p. "/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography subsequent-author-substitute="&#8212;&#8212;&#8212;" entry-spacing="1">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+      <key variable="page-first"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <group>
+        <text macro="author" suffix=" "/>
+        <text macro="year-date" suffix=", "/>
+        <choose>
+          <if type="thesis">
+            <group delimiter="" suffix=".">
```

Comment:
`delimiter=""` doesn't do anything here, so either remove it, or, preferably, use:

```
            <group delimiter=" ">
              <group delimiter=", " suffix=".">
                <text macro="title" font-style="italic"/>
                <text variable="genre"/>
                <text variable="publisher" suffix=" (inédit)"/>
                <text variable="URL" prefix="disponible sur : "/>
              </group>
              <text prefix="[" macro="access" suffix="]"/>
            </group>
```